### PR TITLE
Log request latency and replay buffer size

### PR DESCRIPTION
**Pull request by lmeier**
Note that we'll need some dashboard log work to make this actually visible on the dashboard.

### CODE REVIEW

File: RadarSDK/RadarAPIHelper.m [hunk 1]
```diff
@@ -129,9 +134,17 @@ - (void)requestWithMethod:(NSString *)method
...
                     res = (NSDictionary *)resObj;
 
-                    [[RadarLogger sharedInstance]
-                        logWithLevel:RadarLogLevelDebug
-                             message:[NSString stringWithFormat:@"📍 Radar API response | method = %@; url = %@; statusCode = %ld; res = %@", method, url, (long)statusCode, res]];
+                    if (params && [params objectForKey:@"replays"]) {
+                        NSArray *replays = [params objectForKey:@"replays"];
+                        [[RadarLogger sharedInstance]
+                            logWithLevel:RadarLogLevelDebug
+                                 message:[NSString stringWithFormat:@"📍 Radar API response | method = %@; url = %@; statusCode = %ld; res = %@; latency = %f; replays = %lu", method, url, (long)statusCode, res, latency, (unsigned long)replays.count]];
```
Comment (jsani-radar) [inline RadarSDK/RadarAPIHelper.m:141]: add units for `latency`?

File: RadarSDK/RadarAPIHelper.m [hunk 2]
```diff
@@ -78,7 +78,12 @@ - (void)requestWithMethod:(NSString *)method
                 configuration.timeoutIntervalForResource = 10;
             }
 
+            NSDate *requestStart = [NSDate date];
+
             void (^dataTaskCompletionHandler)(NSData *data, NSURLResponse *response, NSError *error) = ^(NSData *data, NSURLResponse *response, NSError *error) {
+                NSTimeInterval latency = [requestStart timeIntervalSinceNow] * -1;
```
Comment (jsani-radar) [inline RadarSDK/RadarAPIHelper.m:84]: didn't the multiplication for a bit, maybe add a comment that timeIntervalSinceNow returns a negative number?

File: RadarSDK/RadarAPIHelper.m [hunk 3]
```diff
@@ -78,7 +78,12 @@ - (void)requestWithMethod:(NSString *)method
...
+
+
```
Comment (jsani-radar) [inline RadarSDK/RadarAPIHelper.m:86]: nit: extra `\n`

**Review by jsani-radar — COMMENTED** [review]
lgtm, small comments

adding @corypisano and @palmakkar for 👀

File: RadarSDK/RadarAPIHelper.m (hunk 1 above)
Comment (lmeier) [reply RadarSDK/RadarAPIHelper.m:141]: Hmmm, the other values here don't have units, and it sort of breaks the convention going on here. I noted in the comment you suggested that the value is in seconds. Changing this string to something like `latency (s)=` seems non-ideal, but open to switching if you feel strongly.

**Review by nickpatrick — APPROVED** [review]

**Pull request merged**